openupgrade_scripts/scripts/account_payment/17.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+_xmlid_renames = [
+    (
+        "account_payment_invoice_online_payment_patch.enable_portal_payment",
+        "account_payment.enable_portal_payment",
```

Comment:
isn't this already taken care of by the [merge in apriori.py](https://github.com/OCA/OpenUpgrade/blob/17.0/openupgrade_scripts/apriori.py#L18)?